### PR TITLE
add pacific targets

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: ceph-volume-ansible-prs/build/build [hunk 1]
```diff
@@ -30,6 +30,8 @@ elif [[ "$ghprbTargetBranch" == "nautilus" ]]; then
     CEPH_ANSIBLE_BRANCH="stable-4.0"
 elif [[ "$ghprbTargetBranch" == "octopus" ]]; then
     CEPH_ANSIBLE_BRANCH="stable-5.0"
+elif [[ "$ghprbTargetBranch" == "pacific" ]]; then
+    CEPH_ANSIBLE_BRANCH="stable-6.0"
```
Comment (ktdreyer) [inline ceph-volume-ansible-prs/build/build:34]: There's no `stable-6.0` branch at this time. v6.0.0alpha tags are currently on the `master` branch.

Comment (liewegas) [reply ceph-volume-ansible-prs/build/build:34]: Should I leave this off until then?  Or merge it now?

Comment (liewegas) [reply ceph-volume-ansible-prs/build/build:34]: I'll use the alpha branch name

**Review by jdurgin — APPROVED** [review]

**Comment by vumrao** [comment]
quincy https://github.com/ceph/ceph-build/pull/1964

**Pull request merged**